lms/services/h_api.py

```diff
@@ -94,20 +94,32 @@ def upsert_user(self, h_user, provider, provider_unique_id):
         except HAPINotFoundError:
             self.create_user(h_user, provider, provider_unique_id)
 
-    def create_group(self, group_id, group_name, creator):
+    def create_group(self, group_id, group_name):
```

Comment:
```suggestion
    def upsert_group(self, group_id, group_name):
```

I think this change has finally brought to the surface some weirdness that's been going on here for a while. The `PUT ` method for groups is actually upsert, not create:

 * https://h.readthedocs.io/en/latest/api-reference/v1/#tag/groups/paths/~1groups~1{id}/put